README.md

```diff
@@ -253,7 +255,7 @@ This section outlines all the flags you can set to customize the data prep pipel
 | `output_path` | str | `input_file_path`'s directory | Any valid directory path | The directory to store the output files |
 | `log_file_path` | str | `output_path`/logs.log | Any valid file path | The file to save the logs in, this will save the date and time, git commit hash, input arguments and metrics associated with the dataset. |
 | `overwrite_output_path` | bool | False | Include flag for True, no arguments | Permission to delete and overwrite files in `output_path`. |
-| `pretrained_tokenizer` | str | None | Valid tokenizer key from Huggingface | The pretrained tokenizer to be used, loaded using transformers.AutoTokenizer.from_pretrained(args.pretrained_tokenizer), in lieu of a `tokenizer_class`, `vocab_file` and `merges_file`. |
+| `pretrained_tokenizer` | str | None | Valid tokenizer key from Huggingface | The pretrained tokenizer to be used, loaded using transformers.AutoTokenizer.from_pretrained(args.pretrained_tokenizer), in lieu of a `tokenizer_class`, `vocab_file` and `merges_file`. </br> </br> **_Note_**: Llama models/tokeniers are gated, visit the [Llama2 Model Card](https://huggingface.co/meta-llama/Llama-2-7b-hf) to request access!|
```

Comment:
Great addition I lean towards just suggesting using an open non gated llama model? I don't see any issues with this, as they are publicly available 


https://huggingface.co/daryl149/llama-2-7b-chat-hf

Reply:
Also please update this to say that you can also load from a local file path, including a path to a checkpoint where the tokenizer is usually saved? Or any local file path where tokenizer is saved

Maybe link to the documentation for this load function? 
https://huggingface.co/docs/transformers/v4.39.3/en/model_doc/auto#transformers.AutoTokenizer.from_pretrained

Reply:
Both great calls, will add now!